web3_proxy/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(build_hasher_simple_hash_one)]
```

Comment:
What requires this?

Reply:
async-stripe requires it, there's many dependencies. lmk if you want to remove this, it would require a deeper rework of the stripe api though (definitely would take 1-2 full days to separate the parts we're interested in)

Reply:
I'm fine with it. Maybe just add a comment saying what it's for (since it's a lib and not our own code like the other features)